stories/src/examples/DrawToolWithMarkerClusterGroup.tsx

```diff
@@ -155,21 +169,21 @@ const DrawToolWithMarkerClusterGroup: React.FC<Props> = ({
               await getMarkerGroup(layer)
               bindDistanceAndAreaToTooltip(layer)
             }}
-            onDelete={(layersInEditMode) => {
+            onDelete={(layersInEditMode: Array<ExtendedLayer>) => {
```

Comment:
Can't TypeScript infer this type?

Reply:
Apparently not... At least not in my IDE